addons/storyshots/storyshots-core/stories/storyshot.snapshotWithOptionsFunction.test.js

```diff
@@ -0,0 +1,8 @@
+import path from 'path';
+import initStoryshots, { snapshotWithOptions } from '../src';
+
+initStoryshots({
+  framework: 'react',
+  configPath: path.join(__dirname, '..', '.storybook'),
+  test: snapshotWithOptions(() => ({})),
```

Comment:
What does this example test?

Reply:
It tests that when you give it a function it still works (by providing options via the function). Without implementing the function/object detection this would fail